lib/personalAccessKey.ts

```diff
@@ -245,11 +246,13 @@ export async function updateConfigWithAccessToken(
     parentAccountId,
     env: accountEnv,
   });
-  writeConfig();
+  if (!CLIConfiguration.isActive()) {
```

Comment:
Why do we only want to write to the config here when the new version isn't active?

Reply:
In the new version, we write the config directly in `CLIConfiguration`.

Reply:
We may want to add a quick comment in here to explain that, just so that someone who stumbles onto this will understand